example/testapp/src/Makefile

```diff
@@ -1,7 +1,23 @@
 BIN_NAME=serv
+export GOPATH=$(realpath $(dir $(realpath $(lastword $(MAKEFILE_LIST)))).. )
+export GOROOT=$(realpath $(shell which go)/../..)
 
-bin: main.go
-	go build -o $(BIN_NAME)
+
+$(BIN_NAME): main.go
+	go build -i -o $(BIN_NAME)
```

Comment:
Yeah, usually I just run `go get` on sqlite which locally installs it but the `-i` flag is a good option for people who don't know about that.